scripts/dockerhost/cleanup-dockerhosts

```diff
@@ -14,5 +13,4 @@ do
     echo "Cleaning $hostname"
     sudo docker exec $hostname service docker stop
     sudo docker rm -f $hostname
-    sudo ip link delete $i-ext
```

Comment:
is this not needed anymore? This corresponds to host container's veth pair, right?